code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -387,10 +387,15 @@
 		ticker.mode.thralls[ref] = list(H.mind)
 	else
 		ticker.mode.thralls[ref] += H.mind
+	var/datum/objective/protect/new_objective = new /datum/objective/protect
+	new_objective.owner = H:mind
```

Comment:
Not sure what to do differently to convert from : to typepaths, I used apprentice code as an example for this.